### PR TITLE
[2.5] Add December K8s patches

### PR DESCRIPTION
**Pull request by rayandas**
Related: https://github.com/rancher/rancher/issues/35799

Adds:
1.20.14-rancher1-1

### CODE REVIEW

File: rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -5998,10 +5998,10 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
-		// Enabled out of band post v2.5.11
-		"v1.20.13-rancher1-1": {
+		// Enabled in v2.5.12
```
Comment (pennyscissors) [inline rke/k8s_rke_system_images.go]: I've been told we will be doing OOB KDM releases for both 2.5 and 2.6, so this should be `Enabled out of band post v2.5.11”`

Comment (rayandas) [reply rke/k8s_rke_system_images.go]: It's fixed now.

Comment (rayandas) [reply rke/k8s_rke_system_images.go]: It's fixed now.

**Review by snasovich — APPROVED** [review]

**Comment by snasovich** [comment]
Approved based on @PennyScissors comment addressed and him confirming yesterday there are no other changes necessary.

**Pull request merged**